adlfs/multithread.py

```diff
@@ -69,9 +69,10 @@ class ADLDownloader(object):
     adlfs.transfer.ADLTransferClient
     """
     def __init__(self, adlfs, rpath, lpath, nthreads=None, chunksize=2**28,
-                 blocksize=2**22, client=None, run=True, overwrite=False):
+                 blocksize=2**22, client=None, run=True, overwrite=False,
+                 verbose=True):
         if not overwrite and adlfs.exists(rpath):
```

Comment:
I am confused by this logic. Shouldn't it be checking for the existence of the local file? Why are the checks inverted for upload and download based on the errors being raised.